### ISSUE TITLE
Gibberbot flashes/blinks on and off

### ISSUE THREAD

**Issue by earthmeLon** [guardianproject/ChatSecureAndroid#183]
_First of all, thanks again for this.  You've probably already saved people's lives in the process._

My Nexus 7 was recently upgraded to Jelly Bean (4.2/3.1.10) and I started experiencing an issue with Gibberbot that would make the 'Buddy List' and "Chat' flash black/normal very quickly.

The system menu buttons (back/home/menu) are accessible.  After pressing the menu button, the 'menu overlay' also flashes with the screen (but it's different, so I'm noting that)

Even after uninstalling and re-installing Gibberbot, I continue to experience the issue.  The only solution for me has been to Force-Stop the application and clear the cache, while preserving the local data.

So, in summary, about twice a day I have to force-stop and reset the cache for Gibberbot in order for me to use it properly.
